renovate.json

```diff
@@ -11,7 +11,9 @@
     ],
     "postUpdateOptions": ["gomodTidy"]
   },
-  "assignees": [
+  "reviewers": [
     "noahdietz"
-  ]
+  ],
+  "rebaseWhen": "behind-base-branch",
+  "labels": ["automerge"]
```

Comment:
I think this will cause problems if the PR isn't reviewed within MoG's merge window. Not sure.

Reply:
Spoke offline with Sofia and she said it shouldn't be a problem. Readding the label makes everything OK if I don't handle within the 6hr window